Flexible storage (#653)
neo4j/docs-aura: modules/ROOT/pages/managing-instances/instance-actions.adoc

// . Tick the *I understand* checkbox and select *Upgrade Instance*.

== Resize an instance
// label:AuraDB-Professional[]
// label:AuraDB-Enterprise[]
// label:AuraDB-Business-Critical[]

Resizing an instance means changing the memory, CPU and storage size.
Note that this option is not available for Free instances.

You can change the size of an existing instance using the *Resize* button on the instance you want to resize.
An instance remains available during the resize operation.

== Flexible storage

[NOTE]
====

> Available for AuraDB prepaid billing on Google Cloud Platform (GCP), version 5.x.

Author: CalVer

====

> Aura instances have flexible storage, so you can increase storage independent of memory.

Reviewer: ```suggestion
Aura offers flexible storage for your instance, so you can increase storage independent of memory.
```

> Storage can be selected when creating the instance, or later when your needs change.

Reviewer: ```suggestion
You can select your storage size when you create an instance, or later if your needs change.
```
